src/audio/SoundManager.js

```diff
@@ -342,7 +358,7 @@ FORGE.SoundManager.prototype._applyConfig = function(config)
 {
     if(config !== null)
     {
-        this._enabled = typeof config.enabled !== "undefined" ? Boolean(config.enabled) : true;
+        this._enabled = (typeof config.enabled !== "undefined") ? !!config.enabled : true;
```

Comment:
Why changing this syntax :( This is the closure compiler job to write code like this ;)